app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

```diff
@@ -39,6 +39,30 @@ You will only have to make loan repayments once you’re earning.
 
 $student-finance-calculator$
 
+## If you come from outside England
+
+You’ll need to contact your country’s student finance body to find out about your eligibility for funding if you live in Wales, Scotland, or Northern Ireland.
+
+Find out about:
+
+* [student finance in Wales](https://www.studentfinancewales.co.uk/)
+* [student finance in Scotland](https://www.saas.gov.uk/)
+* [student finance in Northern Ireland](https://www.studentfinanceni.co.uk/)
+
+Or, you'll need to contact the education authority if you live in Jersey, Guernsey, or the Isle of Man.
+
+Find out about:
+
+* [funding in Jersey](https://www.gov.je/Working/Careers/16To19YearOlds/EnteringHigherEducation/FinancingHigherEducationCourses/FundingDegreeProfessionalQualifications/Pages/index.aspx)
+* [funding in Guernsey](https://www.gov.gg/article/152744/Policies)
+* [funding in the Isle of Man](https://www.gov.im/student-grants)
+
+### Non-UK citizens
+
+If you’re not a UK citizen, you are unlikely to get help funding your training unless you have permission to [live permanently in the UK](https://www.gov.uk/browse/visas-immigration/settle-in-the-uk), for example under the EU Settlement Scheme, or are training to teach languages or physics. You will also need to meet some other eligibility requirements.
+
+[Find out about funding for non-UK citizens](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
+
 ## Check your eligibility for a scholarship or bursary
```

Comment:
this heading feels a bit chunky @EllieNodder (particularly give you have to go elsewhere for the info) - worth shortening?

Reply:
worth getting these changes live now and thinking about tweaking this after?